das/timeout_wrapper.go

```diff
@@ -41,9 +56,13 @@ func (w *TimeoutWrapper) Store(ctx context.Context, message []byte, timeout uint
 		<-deadlineCtx.Done()
```

Comment:
It looks like this will cancel `deadlineCtx` immediately when `ctx` has been canceled.  I don't think that is the desired behavior.

Reply:
I think we are okay with cancelling the `deadlineCtx` if its parent context gets canceled, the parent being the system's background context.

This code is a little convoluted and will be changed when I move the timeout for stores inside the `Aggregator` rather than doing it from the outside. As far as I can tell
```
		go func() {
			<-deadlineCtx.Done()
			cancel()
		}()
```
doesn't actually do anything beyond silence the linter warning that the cancellation function is leaked.